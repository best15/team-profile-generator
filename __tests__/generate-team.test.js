const Employee = require('../lib/employee');
const Manager = require('../lib/manager');
const Engineer = require('../lib/engineer');
const Intern = require('../lib/intern');

const generateTeam = require('../lib/generate-team.js');

let employees = [];

// employees = new Manager("Manager", "01", "manager@email.com");

describe("generate-team", () => {

    it("generates empty team cards when no employee provided", () => {
        expect(generateTeam(employees)).toMatchSnapshot();

    });

});