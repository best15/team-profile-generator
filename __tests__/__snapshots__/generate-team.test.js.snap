
exports[`generates empty team cards when no employee provided 1`] =
`
"<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/css/bootstrap.min.css">
    <title>My Team</title>
</head>
<body>
    <header class="mb-5">
        <nav class="navbar  bg-secondary ">
            <div class="container-fluid  justify-content-center">
                <h1 class="mb-3 text-center text-light">My Team</h1>
            </div>
        </nav>
    </header>
    <section class="container-fluid">
        <div class="row justify-content-center ">
            <div class="card col-sm-10 col-md-6 col-lg-2 mr-2 mb-3 p-0 ">
                <div class="card-header bg-primary">
                    <h4>sdfasdf</h2>
                    <p>sdfasdfs</p>
                </div>
                <ul class="list-group list-group-flush">
                    <li class="list-group-item">ID:</li>
                    <li class="list-group-item">Email:</li>
                    <li class="list-group-item"></li>
                </ul>
            </div>
        </div>
    </section>
</body>
</html>"
`;